scripts/build/build_node/Platform/Linux/install-ubuntu-build-tools.sh

```diff
@@ -74,6 +74,10 @@ then
     then
         # Ubuntu 22.04 already has an acceptable version of cmake
         echo "Ubuntu 22.04's cmake package already at version $CMAKE_VER"
+    elif [ "$UBUNTU_DISTRO" == "noble" ]
+    then
+        # Ubuntu 24.04 already has an acceptable version of cmake
+        echo "Ubuntu 24.04's cmake package already at version 3.28"
```

Comment:
Shouldn't this mention that Ubuntu 24.04 CMake packages is greater than the `CMAKE_VER` variable (which is 3.22.1) instead of hardcoding the 3.28 version string.

Reply:
Cleared up the messaging